src/js/components/Form/Form.js

```diff
@@ -233,6 +234,14 @@ const Form = forwardRef(
                 // which probably came from a submit
                 setErrors(prevErrors => ({ ...prevErrors, ...nextErrors }));
                 setInfos(prevInfos => ({ ...prevInfos, ...nextInfos }));
+
+                // give user access to errors that have occurred on validation
+                if (onValidate) {
+                  const validation = {};
+                  validation.error = { ...errors, ...nextErrors };
+                  validation.info = { ...infos, ...nextInfos };
+                  onValidate(validation);
```

Comment:
It seems a bit inconsistent to use the `event` for `onSubmit` but not for `onBlur`.

Reply:
I would agree. Updating `onBlur` to use the event as well.